mypy/test/testsemanal.py

```diff
@@ -20,20 +20,22 @@
 # Semantic analyzer test cases: dump parse tree
 
 # Semantic analysis test case description files.
-semanal_files = ['semanal-basic.test',
-                 'semanal-expressions.test',
-                 'semanal-classes.test',
-                 'semanal-types.test',
-                 'semanal-typealiases.test',
-                 'semanal-modules.test',
-                 'semanal-statements.test',
-                 'semanal-abstractclasses.test',
-                 'semanal-namedtuple.test',
-                 'semanal-typeddict.test',
-                 'semenal-literal.test',
-                 'semanal-classvar.test',
-                 'semanal-python2.test',
-                 'semanal-lambda.test']
+semanal_files = [
```

Comment:
This change is important to me, because I use `cmd + #` to comment out all the files and uncomment only a single one I am interested in.

With the previous format it is impossible to do.

Btw, is there any more useful way to run all tests in a file?

Reply:
`pytest ./mypy/test/testsemanal.py` works well for me. `pytest -k search_phrase` is also really useful

Reply:
Yes, but `pytest ./mypy/test/testsemanal.py` will run all `semanal-*.test` files.
I am asking about a single `.test` file, for example only `semanal-classvar.test`